_includes/list-posts

```diff
@@ -0,0 +1,53 @@
+{% comment %}
```

Comment:
I don't see anything including this file; you okay with it being removed?

Reply:
Yeah I assume its part of the blogging platform (add a comment to this post) that we don't use